# Used in time-stamping the message.
std_msgs/Header header

string arm_state_id
string gripper_state_id

JointState joint_state
GripperState gripper_state
